js/viewer/annotation/annotation-svg.js

```diff
@@ -35,17 +36,17 @@ function AnnotationSVG(parent, id, imgWidth, imgHeight, scale, ignoreReferencePo
     }
 
     // Create a new path/rect/circle object for users to draw
```

Comment:
Turn this comment into proper jsdoc and then explain the function and its parameters as best as you can.

Reply:
I don't see a jsdoc comment here.